### ISSUE TITLE
Mouse stops working when vmware guest is enabled

### ISSUE THREAD

**Issue by con-f-use** [NixOS/nixpkgs#102388]
In a NixOS 20.09 guest on a VMWare Windows host, with dwm as my window manager, the mouse cursor neither moves nor registers clicks when `virtualisation.vmware.guest.enable = true`.

**To Reproduce**
1. Install NixOS 20.09 in VMWare Workstation Player 16 VM (from the minimal ISO)
2. Enable xserver and dwm and `virtualisation.vmware.guest.enable` in `configuration.nix`
3. Run `startx` as normal user with a minimal `~/.xinitrc`

@joamaki, can you have a look?

**Comment by kjeremy** [comment]
I can reproduce this. It does work if you login as `root` but fails if you log into a normal user account. It also fails in `gdm`.

**Comment by con-f-use** [comment]
Oh boy, even in the grown-up DM. That's bad.

**Comment by bobhenkel** [comment]
Did this fix the issue for you? @con-f-use ? I'm having issues my self. I also posted in reddit to see if maybe I didn't do something correctly.  https://www.reddit.com/r/NixOS/comments/ooegng/vmware_fusion_nonroot_user_mouse_not_working_with/

**Comment by kjeremy** [comment]
@bobhenkel  try the PR I just linked. My mouse problems went away when I compiled the kernel with CONFIG_PS2_VMMOUSE.

**Comment by bobhenkel** [comment]
@kjeremy Thank you it works! This will be great to have.  This might be a silly question, but is open-vm-tools 11.3.0 required for this to work?  I didn't have time to try 11.2.5 on the stable channel.  

For anyone reading this the name value is MOUSE_PS2_VMMOUSE, might save you one failed kernel build before you figure it out.  Here's my nix config that I used to allow non-root user to use the mouse in Xorg running on vmware fushion on MacOS -> https://gist.github.com/bobhenkel/fdd87fe38a60a38e8367722ac88db70d#file-configuration-issue-fixed-nix.  This config was a "get it to work" attempt with no regard to what the ramifications of how I went about it.  My next step is to figure out what it all really means and see if I went about it correctly.
